server/default_config.py

```diff
@@ -101,7 +101,7 @@
       # data_locator / s3 / region_name.
       tiledb_ctx:
         sm.tile_cache_size:  8589934592  # 8GiB
-        py.init_buffer_bytes: 1073741824  # 1GiB
+        py.init_buffer_bytes: 536870912  # 512MiB
```

Comment:
Is this the recommended configuration for the server? As far as I can remember, this configuration file lives in the single-cell-infra repo (see [here](https://github.com/chanzuckerberg/single-cell-infra/blob/main/terraform/modules/hosted-cellxgene/common/customize-prod/config.yaml)) and I am not quite sure what this python file is for, so we might want to move this to the other file.

Reply:
This is the default used when run as a stand-alone (development) server on your own host/laptop.  Ie, from the CLI.

The deployed config is indeed elsewhere, and should be tuned for the deployment environment.  While the Python buffer size is likely in-common, the tile cache will vary by host

Reply:
That makes sense, thanks!